### DISCUSSION TITLE
Access to the group

### DISCUSSION THREAD

**Discussion by BadrYoubiIdrissi** [Rim-El-Ballouli/entretien-naturalisation-francaise#4] [Q&A] [answered]
Dear maintainers,

Thank you so much for this very helpful resource ! My interview is in two weeks and i would love to get access to the anki flash cards!

Thank you so much !

Best regards.

**Comment by Rim-El-Ballouli** [comment, marked as answer]
Hello,

Your access has been granted. Good luck with your interview!

Regards
